wrappers/services_test.go

```diff
@@ -33,6 +33,7 @@ import (
 
 	"github.com/snapcore/snapd/dirs"
 	"github.com/snapcore/snapd/gadget/quantity"
+	"github.com/snapcore/snapd/overlord/servicestate/resources"
 
```

Comment:
the linter says there should not be an empty line between these packages

Reply:
I don't know why my linter shows nothing, i'll fix it